economy/static/economy/css/economy_home.css

```diff
@@ -0,0 +1,89 @@
+.card {
```

Comment:
Preferably move this to a `default-card` class component in the main.css file in the `internal` module, otherwise add a module prefix